SimG4Core/Application/interface/EventAction.h

```diff
@@ -36,6 +36,9 @@ class EventAction : public G4UserEventAction {
     m_trackManager->addTrack(iTrack, inHistory, withAncestor);
   }
 
+  bool trackExists(unsigned int id) const { return m_trackManager->trackExists(id); }
+  TrackWithHistory* getTrackByID(unsigned int id) const { return m_trackManager->getTrackByID(id); }
```

Comment:
`const` function returning pointer-to-non-`const` is in general bad. Can this be made either to return `const TrackWithHistory *`, or be made non-`const`, or are both constrained by Geant4 or other SIM code?